src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/recordings/batch_help_full.txt

```diff
@@ -365,23 +361,22 @@ Global Arguments
                                     information and examples.
     --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
 
-(env) C:\Code\Forks\azure-cli>cmd /c az batch application package --help 
+(env) D:\GitHub\azure-cli\src\command_modules\azure-cli-batch\azure\cli\command_modules\batch\tests>cmd /c az batch application package --help 
 
 Group
-    az batch application package: Manage your Batch application packages.
+    az batch application package: Manage Batch application packages.
 
 Commands:
-    activate: Activates the specified application package. This step is unnecessary if the package
-              has already been successfully activated by the 'create' command.
-    create  : Create an application package record and activate it.
+    activate: Activates a Batch application package.
```

Comment:
We seem to have lost some of the documentation here too - unless this was specifically removed?

Reply:
Ignore this - the help file was updated out of band for consistency across command modules.